testsuite/tests/load_path/test/driver.ml

```diff
@@ -0,0 +1,4 @@
+#cd "test"
+#remove_directory ""
```

Comment:
I am not sure what is the purpose of this line? As far as I can see this is a no-op and the test passes without it.

Reply:
I think that may have been an accidental hangover from an earlier version while I was battling a bit to set the repro up in ocamltest! I've removed it - the test both fails before without it and passes after the bug fix.